routes/index.js

```diff
@@ -548,11 +553,19 @@ module.exports = function (passport) {
             }
             flag = false;
           }
-          res.render('userManagementBadge.njk', { authenticated: true, user: userInfo.dataValues, allStations });
+          
+          //put most recent events first
+          if (Array.isArray(log)) {
```

Comment:
I wonder if we should modify the `getEvents` function to order by createdDate when it is doing the search. The reason is if there are a lot of events then it becomes costly to sort in the route. I think if we add this to the query parameters it will return them ordered by createdDate:
```
order: '"createdAt" DESC'
```
source: https://stackoverflow.com/questions/20718534/sort-sequelize-js-query-by-date